image.mk

```diff
@@ -68,24 +71,17 @@ deploy-controller:
 
 # Build controller image in dev environment with Golang
 controller-dev: build-base-image
-	$(CONTAINER_TOOL) build . -t ${CONTROLLER_IMAGE_TAG} -f Dockerfiles/Dockerfile.dev --target controller --build-arg BASEIMAGE=$(BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE)
+	$(CONTAINER_TOOL) build --pull . -t ${CONTROLLER_IMAGE_TAG} -f Dockerfiles/Dockerfile.dev --target controller --build-arg BASEIMAGE=$(BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE)
```

Comment:
For the 'dev' targets, should the BASEIMAGE arg be 'CSM_BASEIMAGE'?
Also 'build-base-image' has to be removed.